test/test-against-regression.test.ts

```diff
@@ -0,0 +1,27 @@
+import { FrontendStack } from "../main";
+import { PostsStack } from "../main";
+import { Testing } from "cdktf";
+
+describe("Tests against regression of synth output", () => {
+    it("has no change in FrontendStack Configuration", () => {
+        const app = Testing.app();
+        const stack = new FrontendStack(app, "test-frontend", {
+            environment: "development",
+            user: "regression-test",
+            apiEndpoint: "N/A",
+        });
+        const stackSynth = JSON.parse(Testing.synth(stack));
+        delete stackSynth["terraform"]
+        expect(JSON.stringify(stackSynth, null, 2)).toMatchSnapshot();
```

Comment:
Is there a reason to stringify the stack? Snapshots should be able to compare objects as well (and print a nice diff if they differ) – Feel free to ignore, I'm just curious :)

Reply:
For some reason that had not occurred to me! Good to know 😁